app/routes/admin/sessions/list.js

```diff
@@ -0,0 +1,108 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    switch (this.get('params.sessions_state')) {
+      case 'pending':
+        return this.i18n.t('Pending');
```

Comment:
@niranjan94 @harshitagupta30 a doubt , why can't this be return this.i18n.t(this.get('params.sessions_state')) instead of switch case, thank you !